backends/vulkan/runtime/graph/ops/impl/UnaryOp.cpp

```diff
@@ -114,6 +114,17 @@ float get_val_or_inf(ComputeGraph& graph, const ValueRef& val, bool max) {
         "hardshrink");                                                   \
   }
 
+#define DEFINE_LEAKY_RELU_FN(op_name)                                    \
```

Comment:
why do we need a macro that is only used once?

Reply:
I couldn't use other macros (DEFINE_ACTIVATION_FN for instance) since the func signature is different. Also, I thought maybe other activations (like PReLU) could use the same macro in the future.